Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+  ∃(x)(Sx^(Tx^~Wx)) 
+  S=Is Sally 
+  T=Has an arm on Tuesday 
+  W=Has an arm on Wednesday
+  (b) Every liver has some cell as part at all times it exists.
+  ∀(x)(Lx->∃(y)(Cy^Pyx))
+  L=Is a liver 
+  C=Is a cell 
+  P=_is a part of_
+  
+  (c) John was a child, then an adult, then a senior.
+∃t1∃t2∃t3 (C (j, t1) ∧ A (J, t2) ∧ S(J, t3) ∧ E (t1, t2) ∧ E (t1, t3))
+j = John
+E xy = being earlier than
+C (x, t) = being a child at t
+A (x, t) = being an adult at t
+S (x, t) = being a senior at t
+
+  (d) Goofus and Gallant have been married for three years; for each day of that span, it is true to assert they are married.
+  
+∀t(D(t) ∧ Y(t)→(M(g1,t) ∧ M(g2,t)))
+M(x, t) = being married at t
+Y(t) = belongs to 3 year span 1
+g1 = Goofus
+g2 = Gallant
+D(t) = t is a day
+
+```
+
+5. Using BFO and RO, model the following scenario: the content of an rdf file is represented in two serializations - one in Turtle, one in XML - which are sent from one computer to two distinct computers on the same network.   
+
+Content is an instance_of generic dependent continuant
+File is an instance_of generic dependent continuant
+File 1 Turtle implements content 1
+File 2 XML implements content 1
+Files 1 and 2 are output of Computer 1
+Computer 1 is bearer of File 1 and File 2
+Computer 1 is an instance_of material entity
+Computer 1 sends output to Computers 2 and 3
+File 1 is input of Computer 2
+Computer 2 receives input from Computer 1
+Computer 2 is an instance_of material entity
+File 2 is input of Computer 3
+Computer 3 receives input from Computer 1
+Computer 3 is an instance_of material entity
+Computer 1 2 and 3 are part of Network 1
+Network 1 is an instance_of Network
+Network is_a object aggregate
+
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  Generically Dependent Continuant
+  (b) Chair of the UB Philosophy Department
+  Role
+  (c) SARS-CoV-2
+  Object
+  (d) Mexico City
+  Object Aggregate
+  (e) The trunk of a minivan
+  Fiat Object Part
+  (f) Occupation
+  Role
+  (g) Ocean
+  Object Aggregate
+  (h) Lake
+  Object Aggregate
+```
+
+7. True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  True. A material entity has some portion of matter as proper or improper continuant part. These parts don't need to be material. For example thoughts are immaterial but can be understood as a part of the material person. 
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  False. An immaterial entity is immaterial, so it may be concretized in a material entity or otherwise but may not have it as part. For example, a thought may be an immaterial entity which has a material entity as a subject, but that subject is either not material (i.e is a representation of a material thing) or is not a part of that thought.
```

Comment:
Somewhat weird. First, “concretizes”is “a relationship between a **specifically dependent continuant** and a **generically dependent continuant**”, so it is strange to say an immaterial entity (as an instance of Independent Continuant) is concretized in a material entity (as an instance of Independent Continuant).

Second, if a subject is a material entity as you said, then why is this material entity is not material?

Reply:
Fair point on the "concretizes" use. I didn't mean to invoke that specific usage, so I replaced it with "represents" to convey the point I was trying to make (that we could represent something immaterial materially through some, usually artistic, process).
To answer your second point, the idea was that a material entity could not enter a thought as a material entity. If I think about a table, I could not literally have a material table in the thought. This would be the equivalent of that table appearing literally inside my brain. Instead, I have in mind the idea that what is present in my thought is, instead, the immaterial representation of a material object.